dotcom-rendering/src/components/ExpandableMarketingCardWrapper.importable.tsx

```diff
@@ -161,6 +163,7 @@ export const ExpandableMarketingCardWrapper = ({ guardianBaseURL }: Props) => {
 					isExpanded={isExpanded}
 					setIsExpanded={setIsExpanded}
 					setIsClosed={setIsClosed}
+					abTestVariant={abTestVariant}
```

Comment:
It's also good that we're now able to track the variant, I'm not sure if it was possible to do this via the `data-link-name` approach.